tests/py/dynamo/models/test_models_export.py

```diff
@@ -225,3 +225,49 @@ def calibrate_loop(model):
             )
             outputs_trt = trt_model(input_tensor)
             assert torch.allclose(output_pyt, outputs_trt, rtol=1e-3, atol=1e-2)
+
+
+@unittest.skip(
```

Comment:
Modelopt should be added to the test requirements.txt and the tests should disable if its not installed